src/ProviderImplementations/Anthropic/AnthropicProvider.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace WordPress\AiClient\ProviderImplementations\Anthropic;
+
+use RuntimeException;
+use WordPress\AiClient\Providers\AbstractProvider;
+use WordPress\AiClient\Providers\Contracts\ModelMetadataDirectoryInterface;
+use WordPress\AiClient\Providers\Contracts\ProviderAvailabilityInterface;
+use WordPress\AiClient\Providers\DTO\ProviderMetadata;
+use WordPress\AiClient\Providers\Enums\ProviderTypeEnum;
+use WordPress\AiClient\Providers\ListModelsApiBasedProviderAvailability;
+use WordPress\AiClient\Providers\Models\Contracts\ModelInterface;
+use WordPress\AiClient\Providers\Models\DTO\ModelMetadata;
+
+/**
+ * Class for the Anthropic provider.
+ *
+ * @since n.e.x.t
+ */
+class AnthropicProvider extends AbstractProvider
+{
+    public const BASE_URI = 'https://api.anthropic.com/v1';
+
+    /**
+     * @inheritDoc
+     */
```

Comment:
We should be doing this so we can still track when these concrete methods were created
```suggestion
    /**
     * {@inheritDoc}
     * 
     * @since n.e.x.t
     */
```
Same everywhere else... 😅

Reply:
Updated in https://github.com/WordPress/php-ai-client/pull/39/commits/c6575f5803640b08f6840e547754114f28e70b1f